documentation/man/features/kw-deploy.rst

```diff
@@ -84,9 +84,10 @@ OPTIONS
 -s, \--ls-line:
   List available kernels separated by comma.
 
--u <kernel-name>[,...], \--uninstall <kernel-name>[,...]:
+-u (<kernel-name> | regex:<pattern>)[,...], \--uninstall (<kernel-name> | regex:<pattern>)[,...]:
   Remove a single kernel or multiple kernels; for removing
-  multiple kernels it is necessary to separate them with comma.
+  multiple kernels it is necessary to separate them with comma. A regex pattern
+  can also be passed as input, prefixed with 'regex:'.
```

Comment:
Add some examples in the EXAMPLE section. You can also consider expanding the tutorial part to describe this new regex option.